### PR TITLE
Hotfix problem solution

### PR DESCRIPTION
**Pull request by XinOfWang2001**

### CODE REVIEW

File: Assets/Scripts/MenuScripts/ModifierScript.cs [hunk 1]
```diff
@@ -21,10 +21,10 @@ public void ForwardToSummary()
         summaryScreen.SetActive(true);
     }
 
-    public void SelectKahoot()
+    public void AddSelection(string name)
     {
-        if (!menuManager.modifiers.Contains(1))
-            menuManager.modifiers.Add(1);
-        // else show error message
+        Debug.Log("Henk");
```
Comment (jwruijzenaars) [inline Assets/Scripts/MenuScripts/ModifierScript.cs:26]: Nice debug line ;)

File: Assets/Scripts/MenuScripts/SummaryScript.cs [hunk 2]
```diff
@@ -115,23 +110,18 @@ private void ClassLayoutCheck()
...
-        foreach (var modifier in menuManager.modifiers)
+        foreach(string modifier in menuManager.modifiers)
         {
-            modifierText += modifier.ToString();
-            switch (modifier)
-            {
-                case 1:
-                    {
-                        modifierText += "Kahoot,";
-                        classModifiersText.text = "-Kahoot\n";
```
Comment (jwruijzenaars) [inline Assets/Scripts/MenuScripts/SummaryScript.cs:131]: Where does this get set now?

File: Assets/Scripts/MenuScripts/ModifierScript.cs (hunk 1 above)
Comment (jwruijzenaars) [inline Assets/Scripts/MenuScripts/ModifierScript.cs:26]: Nice debug line

File: Assets/Scripts/MenuScripts/SummaryScript.cs (hunk 2 above)
Comment (JaschavdA) [reply Assets/Scripts/MenuScripts/SummaryScript.cs:131]: Nowhere, i'll go fix it

File: Assets/Scripts/MenuScripts/ModifierScript.cs (hunk 1 above)
Comment (JaschavdA) [reply Assets/Scripts/MenuScripts/ModifierScript.cs:26]: thx

**Pull request merged**